_docs/tasks/traffic-management/ingress.md

```diff
@@ -269,6 +269,344 @@ rules.
    curl -I -k https://$INGRESS_HOST/status/200
    ```
 
+## Configuring RBAC for ingress key/cert
```

Comment:
The key/cert is only used in above section, would it make more sense to put the new content in that section?